apiserver/common/networkingcommon/networkconfigapi_test.go

```diff
@@ -155,8 +155,15 @@ func (s *networkConfigSuite) TestSetObservedNetworkConfigFixesFanSubs(c *gc.C) {
 			InterfaceName: "eth0",
 			InterfaceType: "ethernet",
 			MACAddress:    "aa:bb:cc:dd:ee:f0",
-			CIDR:          "0.10.1.0/24",
-			Address:       "0.10.0.2",
+			Address:       "10.10.10.2",
+			Addresses: []params.Address{
+				{
+					Value: "10.10.10.2",
+					CIDR:  "10.0.0.0/8",
```

Comment:
So is this the core testing change? That the NetworkConfig is returning 10.0.0.0/8 but that should *not* cause us to change the CIDR that we are reporting in line 170/177 ?